win32ss/gdi/eng/stubs.c

```diff
@@ -583,14 +583,24 @@ EngPlgBlt(
 BOOL
 APIENTRY
 EngQueryDeviceAttribute(
-    IN HDEV Device,
-    IN ENG_DEVICE_ATTRIBUTE Attribute,
-    IN VOID *In,
-    IN ULONG InSize,
-    OUT VOID *Out,
-    OUT ULONG OutSize)
+    _In_ HDEV hdev,
+    _In_ ENG_DEVICE_ATTRIBUTE devAttr,
+    _In_reads_bytes_(cjInSize) PVOID pvIn,
+    _In_ ULONG cjInSize,
+    _Out_writes_bytes_(cjOutSize) PVOID pvOut,
+    _In_ ULONG cjOutSize)
 {
     UNIMPLEMENTED;
```

Comment:
What remains unimplemented?

Possible reordered proposal:
```
if (!= QDA_ACCELERATION_LEVEL || < sizeof(DWORD))) 
  return FALSE;

// TODO: Retrieve actual level;
= 0;

return TRUE;
```

Reply:
>What remains unimplemented?
-the slider
-the actual intended functionality

I would rather like to keep it as it is now, because it better shows the flow of how it is supposed to be when someone is fully implementing it later on.

Reply:
I like early returns, but I will trust you that it doesn't apply here.
Nonetheless, according to MS doc, it seems to me `UNIMPLEMENTED;` should be moved after at least the next `if`...

Reply:
Done